app/helpers/validation_helper.rb

```diff
@@ -89,6 +89,18 @@ module ValidationHelper # rubocop:todo Metrics/ModuleLength
     'Other'
   ].freeze
 
+  USER_SELECTABLE_FOLLOW_UP_REASONS = [
```

Comment:
since these are strictly for the follow up stuff I think you should just use it in the necessary component instead of passing it as props throughout